extensions/rocksdb-repos/tests/RepoTests.cpp

```diff
@@ -781,4 +781,69 @@ TEST_CASE("Content repositories are always running", "[TestRepoIsRunning]") {
   REQUIRE(content_repo->isRunning());
 }
 
+std::shared_ptr<minifi::FlowFileRecord> createFlowFileWithContent(core::ContentRepository& content_repo, std::string_view content) {
+  auto flow_file = std::make_shared<minifi::FlowFileRecord>();
+  const auto content_session = content_repo.createSession();
+  const auto claim = content_session->create();
+  const auto stream = content_session->write(claim);
+
+  stream->write(utils::as_span<const std::byte>(std::span(content));
```

Comment:
The CI fails because of a missing parenthesis here.
```suggestion
  stream->write(utils::as_span<const std::byte>(std::span(content)));
```

Reply:
oops, sorry, my fault